content/tutorial/tutorial.md

```diff
@@ -12,185 +12,190 @@ redirect_from:
 
-The `ShoppingList` component above only renders built-in DOM components like `<div />` and `<li />`. But you can compose and render custom React components too. For example, we can now refer to the whole shopping list by writing `<ShoppingList />`. Each React component is encapsulated and can operate independently; this allows you to build complex UIs from simple components.
+`ShoppingList` कौम्पोनॅन्ट ऊपर केवल बिल्ट-इन डोम जैसे `<div />` और `<li />` को रेंडर करता है। लेकिन आप कस्टम React कौम्पोनॅन्ट्स को भी कम्पोज़ और रेंडर कर सकते हैं। उदाहरण के लिए, हम अब पूरी खरीदारी सूची को `<ShoppingList />` लिखकर संदर्भित कर सकते हैं। प्रत्येक React कौम्पोनॅन्ट संकुचित है और स्वतंत्र रूप से काम कर सकता है; यह आपको सरल कौम्पोनॅन्टों से जटिल UI बनाने की अनुमति देता है।
 
-## Inspecting the Starter Code {#inspecting-the-starter-code}
+## स्टार्टर कोड का निरीक्षण {#inspecting-the-starter-code}
 
-If you're going to work on the tutorial **in your browser,** open this code in a new tab: **[Starter Code](https://codepen.io/gaearon/pen/oWWQNa?editors=0010)**. If you're going to work on the tutorial **locally,** instead open `src/index.js` in your project folder (you have already touched this file during the [setup](#setup-option-2-local-development-environment)).
+यदि आप **अपने ब्राउज़र में** ट्यूटोरियल पर काम करने जा रहे हैं, इस कोड को एक नए टैब में खोलें: **[स्टार्टर कोड।](https://codepen.io/gaearon/pen/oWWQNa?editors=0010)**. यदि आप **लोकली** रूप से ट्यूटोरियल पर काम करने जा रहे हैं, तो अपने प्रोजेक्ट फ़ोल्डर में `src/index.js` खोलें ([सेटअप](#setup-option-2-local-development-environment) के दौरान आप इस फ़ाइल को पहले ही छू चुके हैं)।
 
-This Starter Code is the base of what we're building. We've provided the CSS styling so that you only need to focus on learning React and programming the tic-tac-toe game.
+यह स्टार्टर कोड हम जो निर्माण कर रहे हैं उसका आधार है। हमने CSS स्टाइलिंग प्रदान की है ताकि आपको केवल React सीखने और 
+टिक-टैक-टो गेम की प्रोग्रामिंग पर ध्यान देने की आवश्यकता हो।
 
-By inspecting the code, you'll notice that we have three React components:
+कोड का निरीक्षण करके, आप देखेंगे कि हमारे पास तीन React कौम्पोनॅन्ट हैं:
 
 * Square
 * Board
 * Game
 
-The Square component renders a single `<button>` and the Board renders 9 squares. The Game component renders a board with placeholder values which we'll modify later. There are currently no interactive components.
+Square कौम्पोनॅन्ट एक एकल `<button>` और Board को 9 वर्गों रेंडर करता है। Game कौम्पोनॅन्ट प्लेसहोल्डर मानों के साथ एक Board रेंडर करता है जिसे हम बाद में संशोधित करेंगे। वर्तमान में कोई इंटरैक्टिव कौम्पोनॅन्ट नहीं हैं।
 
-### Passing Data Through Props {#passing-data-through-props}
+### Props के माध्यम से डेटा पास करना {#passing-data-through-props}
 
-To get our feet wet, let's try passing some data from our Board component to our Square component.
+बस अपने पैरों को गीला करने के लिए, आइए हमारे Board कौम्पोनॅन्ट से हमारे Square कौम्पोनॅन्ट के कुछ डेटा को पास करने का प्रयास करें।
```

Comment:
बस अपने पैरों को गीला करने के लिए -> 😆

Reply:
revisit the whole line, literal translation is totally funny 🤣

Reply:
I know, when I was translating, I couldn't think of anything else :rofl: 
Any suggestions? 
`बस अपने पैरों को गीला करने के लिए,` => `Kuch suruat krne ke liye`?

Reply:
@saranshkataria any inputs on this?

Reply:
शुरुआत के लिए, आइए hum Board कौम्पोनॅन्ट से हमारे Square कौम्पोनॅन्ट ko कुछ डेटापास करने का प्रयास karte hain